src/main/java/com/minecolonies/achievements/AbstractSizeAchievement.java

```diff
@@ -0,0 +1,76 @@
+package com.minecolonies.achievements;
+
+import net.minecraft.block.Block;
+import net.minecraft.entity.player.EntityPlayer;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.entity.player.EntityPlayer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)